src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -617,17 +617,17 @@ def test_update_webapp_settings_thru_json(self, resource_group):
 
         self.assertEqual(output[0], {
             'name': 's',
-            'value': 'False',
+            'value': 'value',
```

Comment:
Do we have a test case with this JSON format for both app & slot. If not can you add one, so we don't miss this again? Thanks for taking care of this quickly.

Reply:
Added